packages/at_persistence_secondary_server/lib/src/log/commitlog/sync/skip_deletes_strategy.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:at_persistence_secondary_server/src/log/commitlog/commit_entry.dart';
+import 'package:at_persistence_secondary_server/src/log/commitlog/sync/sync_keys_fetch_strategy.dart';
+
+class SkipDeleteStrategy extends SyncKeysFetchStrategy {
+  @override
+  bool shouldIncludeEntryInSyncResponse(
+      CommitEntry commitEntry, int commitId, String regex,
+      {List<String>? enrolledNamespace}) {
```

Comment:
Dont we need call to super.shouldIncludeEntryInSyncResponse()?

Reply:
i kept the method in base class abstract